http://git.savannah.gnu.org/cgit/freetype/freetype2.git/patch/?id=b9880aa0f8f52accc9074334f9e9f962b1b5a8e6

From b9880aa0f8f52accc9074334f9e9f962b1b5a8e6 Mon Sep 17 00:00:00 2001
From: Werner Lemberg <wl@gnu.org>
Date: Mon, 12 Oct 2015 08:13:26 +0000
Subject: [unix] Make MKDIR_P actually work.

* builds/unix/configure.raw: Fix underquoting of `INSTALL' and
`MKDIR_P'.

Problem reported by Dan Liddell <lddll@yahoo.com>.
---
diff --git a/ChangeLog b/ChangeLog
index 067bf74..b02735f 100644
--- a/ChangeLog
+++ b/ChangeLog
@@ -1,3 +1,12 @@
+2015-10-12  Werner Lemberg  <wl@gnu.org>
+
+	[unix] Make MKDIR_P actually work.
+
+	* builds/unix/configure.raw: Fix underquoting of `INSTALL' and
+	`MKDIR_P'.
+
+	Problem reported by Dan Liddell <lddll@yahoo.com>.
+
 2015-10-11  Werner Lemberg  <wl@gnu.org>
 
 	[sfnt] Improve extraction of number of named instances.
diff --git a/builds/unix/configure.raw b/builds/unix/configure.raw
index 5a30c65..f28baff 100644
--- a/builds/unix/configure.raw
+++ b/builds/unix/configure.raw
@@ -70,24 +70,25 @@ AC_SUBST(CC_BUILD)
 AC_SUBST(EXEEXT_BUILD)
 
 
-# Since this file will be finally moved to another directory we make
-# the path of the install scripts absolute.  This small code snippet has
-# been taken from automake's `ylwrap' script.
+# Since these files will be eventually called from another directory (namely
+# from the top level) we make the path of the scripts absolute.
+#
+# This small code snippet has been taken from automake's `ylwrap' script.
 
 AC_PROG_INSTALL
 case "$INSTALL" in
-[\\/]* | ?:[\\/]*)
+[[\\/]]* | ?:[[\\/]]*)
   ;;
-*[\\/]*)
+*[[\\/]]*)
   INSTALL="`pwd`/$INSTALL"
   ;;
 esac
 
 AC_PROG_MKDIR_P
 case "$MKDIR_P" in
-[\\/]* | ?:[\\/]*)
+[[\\/]]* | ?:[[\\/]]*)
   ;;
-*[\\/]*)
+*[[\\/]]*)
   MKDIR_P="`pwd`/$MKDIR_P"
   ;;
 esac
--
cgit v0.9.0.2
